kfs-cam/src/main/resources/edu/arizona/kfs/module/cam/ojb-cam.xml

```diff
@@ -0,0 +1,57 @@
+<descriptor-repository version="1.0">
```

Comment:
Without the override, newBo in KualiGlobalMaintainableImpl.java will be the org/kuali version of AssetRetirementGlobal and run the wrong setAssetForPersist method.